src/Kunstmaan/GeneratorBundle/Generator/KunstmaanGenerator.php

```diff
@@ -203,6 +203,7 @@ protected function generateEntityAdminType(
      *
      * @param BundleInterface $bundle
      */
+    
```

Comment:
don't do code formatting of old code in a feature PR, als think that there should not be spaces here